packages/storybook/src/decorators/withKeytipLayer.tsx

```diff
@@ -10,6 +10,6 @@ export const KeytipLayerWrapper: React.FunctionComponent<{}> = props => {
   );
 };
 
-export const withKeytipLayer = (storyFn: () => React.ReactNode) => {
-  return <KeytipLayerWrapper>{storyFn()}</KeytipLayerWrapper>;
+export const withCompatKeytipLayer = (storyFn: () => React.ReactNode) => {
```

Comment:
I think you forgot to change the name of this to withKeytipLayer? (or need to update references elsewhere)